### PR TITLE
add settings.yml

### PR DESCRIPTION
**Pull request by bobcallaway**
Signed-off-by: Bob Callaway <bcallaway@google.com>

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+repository:
+  # See https://developer.github.com/v3/repos/#edit for all available settings.
+
+  # The name of the repository. Changing this will rename the repository
+  name: wg-securing-software-repos
+
+  # A short description of the repository that will show up on GitHub
+  description: The Securing Software Repositories working group is for and focuses on the maintainers of software repositories, software registries, and tools which rely on them, at various levels including system, language, plugin, extensions and container system. It provides a forum to share experiences and to discuss shared problems, risks and threats.
```
Comment (jchestershopify) [inline .github/settings.yml:8]: Typo: container system -> systems.

**Review by jchestershopify — COMMENTED** [review]
Should @di be an admin too?

**Comment by bobcallaway** [comment]
> Should @di be an admin too?

yes, good catch

**Review by jchestershopify — APPROVED** [review]
LGTM.

**Review by di — APPROVED** [review]

**Pull request merged**